automatically split up statistics queries

lelilia opened the pull request:
pairing with relud 

waiting to write tests until anna confirms that this is the right thing to be splitting queries on

lelilia commented on opmon/monitoring.py:
```diff
@@ -37,6 +38,8 @@
 SCHEMA_VERSIONS = {"metric": 1, "statistic": 2, "alert": 2}
 METRICS_JOIN_KEYS = ["client_id", "submission_date", "build_id", "branch"]
 MAX_DIMENSIONS_PER_METRIC_QUERY = 40
+STATISTICS_JOIN_KEYS = ["client_id", "submission_date", "build_id", "branch"]
```
are those the correct join keys for statistics?

lelilia commented:
not the way to do it so not review yet

lelilia commented:
look at how get_mertric_sql did it ... not dimensions but summary

scholtzan commented:
Sorry, I haven't gotten around to taking a look at this. It's not forgotten, but will have to wait until next week

lelilia commented:
sorry Anna we removed you from review and moved it to draft because we realized at the end of the session that it's not doing what we want so no need for review yet
